### PR TITLE
Events are reported as logs

### PR DESCRIPTION
**Pull request by codeboten**
Events were not being reported, this change fixes that. I've also added tests to ensure attributes are set in tags.

### CODE REVIEW

File: .flake8 [hunk 1]
```diff
@@ -0,0 +1,6 @@
+[flake8]
+ignore =
+  E501  # line too long, defer to black
+  F401  # unused import, defer to pylint
+  W503  # allow line breaks after binary ops, not after
+  E203  # allow whitespace before ':' (https://github.com/psf/black#slices)
```
Comment (ocelotl) [inline .flake8:6]: Is something missing at the very end of this line? :laughing:

File: src/opentelemetry/ext/lightstep/__init__.py [hunk 2]
```diff
@@ -57,6 +57,11 @@ def export(self, spans: typing.Sequence[Span]) -> "SpanExportResult":
                 start_time=_nsec_to_sec(span.start_time),
                 tags=span.attributes,
             )
+            for event in span.events:
+                event.attributes["message"] = event.name
```
Comment (ocelotl) [inline src/opentelemetry/ext/lightstep/__init__.py:61]: Just a bit of semantic confusion here, the "message" of the event is also its "name"?

File: tests/test_trace_exporter.py [hunk 3]
```diff
@@ -114,7 +112,66 @@ def test_export(self, mock_tracer):
...
+    #         name=__name__, context=self._span_context, links=(link,),
+    #     )
+    #     result_spans = self._process_spans([otel_span])
+    #     self.assertEqual(len(result_spans), 1)
+    #     self.assertEqual(len(result_spans[0].references), 1)
+
+    @patch("lightstep.Tracer")
+    def test_resources(self, mock_tracer):
+        """ test resources """
+        # TODO
```
Comment (ocelotl) [inline tests/test_trace_exporter.py:177]:
```suggestion
```

Empty tests are still collected, executed and reported.

File: src/opentelemetry/ext/lightstep/__init__.py (hunk 2 above)
Comment (codeboten) [reply src/opentelemetry/ext/lightstep/__init__.py:61]: Yeah, this is setting the `message` attribute to the name, as logs don't really have the concept of a name

File: tests/test_trace_exporter.py (hunk 3 above)
Comment (codeboten) [reply tests/test_trace_exporter.py:177]: i think that's fine, I will be filling those in before the next release.

File: .flake8 (hunk 1 above)
Comment (codeboten) [reply .flake8]: fixed

**Pull request merged**